### ISSUE TITLE
Focus indicators on main toolbar have inconsistent display

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7604]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.1075
    OS Version      : All
    R Version       : Any

### Steps to reproduce the problem
- Varies by desktop vs server and by platform. The worst case is RStudio Server.
- Get focus to the main toolbar via Help / Accessibility / Focus / Focus Main Toolbar (or use keyboard shortcut)
- If focus not displayed (#6619) tap Shift key to show it

### Describe the problem in detail
- The focus rectangle isn't always symmetrical; on Server, it is missing the top edge completely; on Windows Desktop, the bottom edge is partially cut off, but still visible

<img width="193" alt="screenshot of RStudio Server in Chrome showing focus on first main toolbar button" src="https://user-images.githubusercontent.com/10569626/90550516-ea1cc080-e144-11ea-91d1-7212010757b4.png">

### Describe the behavior you expected
- Focus rectangles should have consistent appearance

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
